packages/docusaurus-init/templates/classic/sidebars.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   someSidebar: {
-    Docusaurus: ['doc1', 'doc2', 'doc3'],
+    Docusaurus: ['getting-started'],
```

Comment:
I suggest we could use the following docs/sidebar:

```js
module.exports = {
  docs: {
    "Docusaurus Tutorial": [
      "Docusaurus Intro", // Basic infos + install steps
      "Create a doc", // + how to add it to  the sidebar
      "Create a page", // jsx + md
      "Create a post",
      "Markdown Features", // basic markdown + some MDX explanations
      "Thank you" // end of tutorial, show what's next?
    ],
    Extras: [
      "Deploy your site", // configure a bit + build + deploy to GH pages?
      "Translate your site", // very basic version, like add "fr" language + put a copy `docs/intro.md` to `i18n/fr/plugin-docs/intro.md` + run `yarn start --locale fr`
      "Manage versions" // just run the versioning cli + restart the site with yarn start
    ]
  }
};
```

Does it make sense to you?

Maybe it's too much doc, don't know, we may remove some sections from the Extras category?

Reply:
Yes, thank you for giving me out this overall structure. I already put some work in the Docusaurus Tutorial part and I think there are too many things in the Markdown Features right now, maybe we can like include some examples in `Create a doc` section and then for the details we can refer to the main documentation site?

For the Extras part, I think Translate your site is a little bit too specific (this is pretty subjective), we can just refer to the main docs.